src/pa/proto/pa.proto

```diff
@@ -179,7 +179,7 @@ message CloseSessionResponse {
 // Device Registration request.
 message RegistrationRequest {
   // Device record. Required.
-  ot_device_id.DeviceRecord device_record = 1;
+  ot_device_id.DeviceData device_data = 1;
```

Comment:
Something we could do is to create name space as follows:

`ot.device` or `ot.device.id`, and later on `ot.device.record`. This is not as important for golang code generation, but it is nice for C++, since we'll end up with the following namespace: `::ot::device`.

Reply:
added a commit to fix. New namespace is `ot`